core/shared/src/main/scala/zio/ConfigProvider.scala

```diff
@@ -154,7 +154,7 @@ object ConfigProvider {
             ZIO
               .fromOption(valueOpt)
               .mapError(_ => Config.Error.MissingData(path, s"Expected ${pathString} to be set in the environment"))
-          results <- Flat.util.parsePrimitive(value, path, name, primitive, ":")
+          results <- Flat.util.parsePrimitive(value, path, name, primitive, ",")
```

Comment:
Maybe it would worth making this a parameter (and keeping `envProvider` a value that calls it with `","` so if others have values with commas in them they can select another separator?

Reply:
Done.